task/06-conditions-n-loops-tasks.js

```diff
@@ -356,7 +477,18 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 10 => '365'
  */
 function toNaryString(num, n) {
-    throw new Error('Not implemented');
+  let remain = num;
+  let resultNum = '';
+  let i = 0;
+  while (num % Math.pow(n, i) !== num) {
+    i++;
+  }
+  let m = i - 1;
+  for (m; m >= 0; m--) {
+    resultNum += Math.floor(remain / Math.pow(n, m));
+    remain = remain % Math.pow(n, m);
+  }
+  return +resultNum;
```

Comment:
We can just use toString method with radix as parameter
https://www.w3schools.com/jsref/jsref_tostring_number.asp